README.md

```diff
@@ -1,2 +1,34 @@
-# docs-first-discovery
-Documentation for the First Discovery Tool
+# First Discovery Tool Documentation
+
+This project contains the content needed to build and deploy the First Discovery Tool Documentation. It is based off of the Fluid `docs-template` project.
+
+# To Build
+
+1. Clone this repository.
+2. From within the project's directory, install DocPad if it isn't already installed: `sudo npm install -g docpad`
+3. Get the required node modules: `npm install`
+4. Run docpad: `docpad run`
```

Comment:
It completed ok but I am unable to reproduce the issue on my desktop. I'll need to get laptop out again to see if I can characterise the problem better.